### ISSUE TITLE
GenerateSerializer models don't work with `required` properties

### ISSUE THREAD

**Issue by zpxp** [dotnet/orleans#8960]
Using the `required` keyword on properties on a class that is decorated with `[GenerateSerializer]` results in compile exceptions

```c#
[GenerateSerializer]
public class SomeClass 
{
	[Id(1)] public required string Name { get; set; }
}
```

```
Orleans.CodeGenerator.OrleansSerializationSourceGenerator/Game.orleans.g.cs(1941,34): error CS9035: Required member 'SomeClass.Name' must be set in the object initializer or attribute constructor.
```

**Comment by ReubenBond** [comment]
Which version of Orleans are you using, @zpxp? This ought to have been fixed by this PR: https://github.com/dotnet/orleans/pull/8595

**Comment by zpxp** [comment]
> Which version of Orleans are you using, @zpxp? This ought to have been fixed by this PR: #8595

We are using `7.2.1`

**Comment by ReubenBond** [comment]
@zpxp this was fixed in [v7.2.2](https://github.com/dotnet/orleans/releases/tag/v7.2.2)
